src/map/battle.cpp

```diff
@@ -5898,6 +5899,13 @@ static int32 battle_calc_attack_skill_ratio(struct Damage* wd, struct block_list
 			skillratio += -100 + 700 * skill_lv + 10 * sstatus->pow;
 			RE_LVL_DMOD(100);
 			break;
+		case IQ_BLAZING_FLAME_BLAST:
+			skillratio += -100 + 2000 + 3800 * skill_lv;
+			skillratio += 10 * sstatus->pow;
```

Comment:
Can you confirm the ratio? otherwise :
```suggestion
			skillratio += 10 * sstatus->pow;	// !TODO: unknown ratio
```